README.md

```diff
@@ -260,52 +261,59 @@ Performs a manual search against the specified assistant. To retrieve recommenda
 
 * `assistantId`: The identifier of the Amazon Q Connect assistant. Can be either the ID or the ARN. URLs cannot contain the ARN.
 * `queryText`: The text to search for.
+* `sessionId`: The identifier of the session. Can be either the ID or the ARN. URLs cannot contain the ARN.
 * `maxResults`: The maximum number of results to return per page.
+* `queryCondition`: The information about how to query content.
 * `nextToken`: The token for the next set of results. Use the value returned in the previous response in the next request to retrieve the next set of results.
 
 #### A few things to note:
 
-* The `assistantId` can be retrieved by using the `ListIntegrationAssociations` API provided by QConnectJS to look up the `assistant` and `knowledge base` that has been configured for Amazon Q Connect. See [ListIntegrationAssociations](#ListIntegrationAssociations) for more information.
+* The `assistantId` can be retrieved by using the `ListIntegrationAssociations` API provided by QConnectJS to look up the `assistant` and `knowledge base` that has been configured for Amazon Q Connect. See [ListIntegrationAssociations](#listintegrationassociations) for more information.
 
 ### Response Syntax
 
 If the action is successful, the service sends back an HTTP 200 response.
 
 ```json
 {
-  "nextToken": "string",
-  "results": [
-    {
-      "document": {
-        "contentReference": {
-          "contentArn": "string",
-          "contentId": "string",
-          "knowledgeBaseArn": "string",
-          "knowledgeBaseId": "string"
-        },
-        "excerpt": {
-          "highlights": [
-            {
-              "beginOffsetInclusive": "number",
-              "endOffsetExclusive": "number"
-            }
-          ],
-          "text": "string"
-        },
-        "title": {
-          "highlights": [
-            {
-              "beginOffsetInclusive": "number",
-              "endOffsetExclusive": "number"
+   "nextToken": "string",
+   "results": [
+      {
+         "data": {
+            "details": { ... },
```

Comment:
any reason to show show the nested structure for these?

Reply:
They are union type, so hard to display.